lib/vyasa_web/live/media_live/media_bridge.ex

```diff
@@ -187,6 +238,43 @@ defmodule VyasaWeb.MediaLive.MediaBridge do
      |> push_hook_events()}
   end
 
+  # @bala set cursor handle event passes sufficient info here.
+  def handle_event(
+        "setCursor",
+        %{
+          "track_order" => track_order,
+          "track_id" => track_id,
+          "tracklist_id" => tracklist_id,
+          "verse_id" => verse_id,
+          "chapter_no" => chapter_no,
+          "source_id" => source_id,
+          "source" => source
+          # value: _value
+        },
+        socket
+      ) do
+    IO.inspect({track_order, track_id, tracklist_id, verse_id, chapter_no, source_id, source},
+      label: "setCursor event received with track_order and track_id"
+    )
+
+    # Do something with the track_order and track_id here, e.g., update the cursor
+
+    send(socket.parent_pid, %{
+      event: :set_cursor_in_tracklist,
+      pid: self(),
+      origin: __MODULE__,
+      tracklist_cursor: track_order,
+      track_id: track_id,
+      tracklist_id: tracklist_id,
+      verse_id: verse_id,
+      chapter_no: chapter_no,
+      source_id: source_id,
+      source: source
```

Comment:
this could be cleaned up by passingthe tracklist around, i had issues with it because of the json encoding stuff, but didn't know that we could jsut do a struct-serialise of sorts and pass it around 

this is a refactoring candidate, but once again we can prioritse that cleanup for later